src/chains.jl

```diff
@@ -333,6 +333,78 @@ Base.last(c::Chains) = last(c.value[Axis{:iter}].val)
 
 Base.convert(::Type{Array}, chn::Chains) = convert(Array, chn.value)
 
+"""
+    start_time(c::Chains)
+
+Retrieve a vector of start times (as `DateTime`)
+from `chain.info`. Assumes that either
+`DateTime` or unix timestamps (`Float64`) are stored in
+`chain.info.start_time`
+"""
+function start_time(c::Chains)
```

Comment:
Maybe one should just use a special `struct` and dispatch on it?

Reply:
I changed the internals here to dispatch on the type of `c.info.start_time` and `c.info.stop_time`.